packages/mobile/src/recipients/recipient.ts

```diff
@@ -140,19 +124,29 @@ export function getRecipientFromAddress(
   return e164PhoneNumber ? recipientCache[e164PhoneNumber] : undefined
 }
 
-export function getRecipientVerificationStatus(
+export const getRecipientVerificationStatus = (
   recipient: Recipient,
   e164NumberToAddress: E164NumberToAddressType
-): RecipientVerificationStatus {
+): RecipientVerificationStatus => {
   if (recipient.kind === RecipientKind.QrCode || recipient.kind === RecipientKind.Address) {
     return RecipientVerificationStatus.VERIFIED
   }
 
-  if (!recipient.e164PhoneNumber) {
-    throw new Error('No recipient e164Number found')
+  if (recipient.e164PhoneNumber) {
```

Comment:
Really small nit but have you heard of the zen-of-python reduce nesting principle? TLDR, look for opportunities to reduce nesting by inverting `if`s. Instead of the bulk of logic being in this if, check the inverse and return UNKNOWN in that case

Reply:
Will implement. Useful tactic, thanks!